wire-ios-utilities/Source/Validation/ZMEmailAddressValidator.swift

```diff
@@ -88,16 +88,11 @@ import UIKit
         let emailScanner = Scanner(string: emailAddress! as String)
         emailScanner.charactersToBeSkipped = CharacterSet()
         emailScanner.locale = Locale(identifier: "en_US_POSIX")
+        let local: String? =  emailScanner.scanUpToString("@")
+        _ = emailScanner.scanString("@")
```

Comment:
why do we need this?

Reply:
to  move the scanner beyond `@`

Reply:
We should have regex here. Not sure why this is done in this specific way.